test/wdio/watch-native-attributes/cmp.test.tsx

```diff
@@ -0,0 +1,22 @@
+import { h } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+describe('watch native attributes', () => {
+  beforeEach(() => {
+    render({
+      template: () => <watch-native-attributes aria-label="myStartingLabel"></watch-native-attributes>,
+    });
+  });
+
+  it('triggers the callback for the watched attribute', async () => {
+    await $('watch-native-attributes').waitForExist();
+
+    const cmp = document.querySelector('watch-native-attributes');
+    expect(cmp.innerText).toBe('Label: myStartingLabel\n\nCallback triggered: false');
+
+    cmp.setAttribute('aria-label', 'myNewLabel');
+    await browser.pause(100);
+
+    expect(cmp.innerText).toBe('Label: myNewLabel\n\nCallback triggered: true');
```

Comment:
I would suggest something like this to leverage the async matchers.
```suggestion
    const $cmp = $('watch-native-attributes')
    await $cmp.waitForExist();

    await expect($cmp).toHaveText('Label: myStartingLabel\n\nCallback triggered: false');

    const cmp = document.querySelector('watch-native-attributes');
    cmp.setAttribute('aria-label', 'myNewLabel');

    await expect(cmp).toHaveText('Label: myNewLabel\n\nCallback triggered: true');
```

I looking into ways to easier retrieve the raw DOM element from the `WebdriverIO.Element` object to allow us to just define the element once.

Reply:
Is the `$` variable prefix the recommended convention when using async queries? Just curious!

Reply:
not really .. but in the context of using `HTMLElement` and `WebdriverIO.Element` objects it seems like a good idea to do but not a convention.